Examples/Scripts/FreeSurferPipelineBatch-long.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+function identify_timepoints
+{
+    local subject=$1
+    local tplist=""
+    local tp visit n
+
+    #build the list of timepoints (sessions)
+    n=0
+    for session in ${PossibleVisits[*]}; do
+        tp="${subject}_${session}"
+        if [ -d "$StudyFolder/$tp" ] && ! [[ " ${ExcludeVisits[*]} " =~ [[:space:]]"$tp"[[:space:]] ]]; then
+             if (( n==0 )); then 
+                    tplist="$tp"
+             else
+                    tplist="$tplist,$tp"
+             fi
+        fi
+        ((n++))
+    done
+    echo $tplist
+}
```

Comment:
I suggest putting this function below the settings.  We want the settings the user should edit to be close to the top of the file.  I don't think they will need to edit the filenames inside that loop, so it seems okay to have the function definition in between the settings and the loop.

Reply:
I'm fine with that as long as this logic is consistent throughout all scripts. Done.